### DISCUSSION TITLE
plugin window position

### DISCUSSION THREAD

**Discussion by fojerhar** [apohl79/audiogridder#934] [General]
Hi
Since i dont know when the opened plugin window position went back to the protools plugin window. If I replace it, and move the PT window the opened plugin jumps back to it.
How should I reset or solve this thing?
![Screenshot 2022-04-03 at 20 06 06](https://user-images.githubusercontent.com/72139943/161441806-aec8d7f7-03e5-4226-ab1b-7b1d0ca303e1.png)
thank you

**Comment by apohl79** [comment]
When ever you change the pro tools window position, the AG window follows. But it looks like the pro tools window width is higher than the AG UI. Is this something new in later PT versions?

You can try to enable to keep the plugin window on top of everything in the AG server settings. That would hopefully render it on top the PT window.

**Reply by fojerhar** [reply]
Hi
Did not helped. But the position changes weirdly if I change the zoom in plugin
https://www.dropbox.com/s/ii88rrg4knt42le/Screen%20Recording%202022-04-04%20at%2016.22.50.mov?dl=0

But this is a small cosmetic issue...
thank you

**Reply by apohl79** [reply]
This will be fixed in the next version.

**Comment by fojerhar** [comment]
HI Andreas
Nowdays I started to have many collapses, can you help me find out what is the issue with me?
https://www.dropbox.com/sh/pryc2pci4tf7864/AADz7zuL__eJ9FPaKVEQfbh8a?dl=0
Here all daw is closed, these are the unresponding, got stucked sandboxes
![Screenshot 2022-04-04 at 18 27 08](https://user-images.githubusercontent.com/72139943/161589425-5f27bb8c-257f-46ab-b571-2df4d6da8b43.png)
thank you
